Sources/PackageCollections/API.swift

```diff
@@ -110,9 +110,12 @@ public protocol PackageCollectionsProtocol {
     ///
     /// - Parameters:
     ///   - reference: The package reference
+    ///   - collections: Optional. If specified, only look for package in these collections. Data from the most recently
+    ///                  processed collection will be used.
     ///   - callback: The closure to invoke when result becomes available
     func getPackageMetadata(
         _ reference: PackageReference,
+        collections: Set<PackageCollectionsModel.CollectionIdentifier>?,
```

Comment:
default to nil?

Reply:
Cannot because this is a protocol. Default to nil in the impl though.

Reply:
Could we make it optional in the protocol by having two overloads? This might help to not immediately break clients which are calling this via the protocol using the old set of arguments.

Reply:
Amended https://github.com/apple/swift-package-manager/pull/3389/commits/e633ba759da01a159496e0332402386bc0a1bdca and undid the change in `SwiftPackageCollectionsTool.swift`.